gef-ui/webpack.common.config.js

```diff
@@ -23,6 +23,13 @@ module.exports = {
       new CssMinimizerPlugin({}),
     ],
   },
+  resolve: {
```

Comment:
In your docs section noting how hoisting works especially with regard to frontend packages etc. like here would be good

Reply:
Separate doc now added [here](https://github.com/UK-Export-Finance/dtfs2/pull/2596/files#diff-0b220ba191203bac20bc8dde2113320ff8bbaeaf016cf66efd7b125c9ab924cd) with some extra notes on the limitations of hoisting.